### PR TITLE
Rename project to NDLAInteractiveBoard

### PR DESCRIPTION
**Pull request by boyum**

### CODE REVIEW

**Review by henriettemoe — APPROVED** [review]

File: scripts/bg-selector.js [hunk 1]
```diff
@@ -51,8 +51,8 @@ export default (function ($, EventDispatcher) {
 
     // Labels for radio buttons
     var radioLabels = [
-      H5PEditor.t('H5PEditor.NDLACoursePresentation', 'setImageBackground', {}),
-      H5PEditor.t('H5PEditor.NDLACoursePresentation', 'setColorFillBackground', {})
+      t('setImageBackground', {}),
```
Comment (otacke) [inline scripts/bg-selector.js:56]: This will break, because `t` is not imported in `bg-selector.js`.

File: scripts/utils.js [hunk 2]
```diff
@@ -29,3 +29,13 @@ export function getLibraryDependencyVersion(libraryName) {
 
   return `${dependency.majorVersion}.${dependency.minorVersion}`;
 }
+
+/**
+ * @param {string} translationKey
```
Comment (otacke) [inline scripts/utils.js]: I think every JSDoc comment for a function should contain one line that explains what the function does, and every parameter and the return value should at least briefly explain what they are used for/what to expect. That's not relevant if one thinks about potentially checking type-safety only (if one just sees JSDoc for that purpose), but the comments/descriptions are actually what I think are useful when you try to understand the code.

I for my part would for instance like to know that `rest` means "Placeholders and values to replace in the text." (https://github.com/h5p/h5p-editor-php-library/blob/2f319eb31a16644bb5e864413926d88f8873954b/scripts/h5peditor-editor.js#L610) ;-)

**Review by otacke — CHANGES_REQUESTED** [review]
Good idea to wrap the translation function as part of changing the machine name (some would preferred separate commits, but I don't think that does really matter in this case). With introducing `t`, you've introduced a bug however. And it is not my call I think, but I'd make the JSDoc comments more useful.

We may need agree on this: In theory, I could have fixed the bug myself quickly, but every project seems to handle this differently, so I stick to comments for now.

File: scripts/bg-selector.js (hunk 1 above)
Comment (boyum) [reply scripts/bg-selector.js:56]: Thanks! Yet another point in favour of typing, though the argument is that I'm not keen-eyed 🥲

File: scripts/utils.js (hunk 2 above)
Comment (boyum) [reply scripts/utils.js]: Thanks for the thorough review! The JSDoc was meant for typing only, and as this is no more than a wrapper function to move the library name out of the code, I didn't see the point of documenting it further. I do agree with you on typing `rest` in a better way, and renaming it.

**Comment by boyum** [comment]
Thanks for your review, @otacke!

> some would preferred separate commits, but I don't think that does really matter in this case

I agree that separate commits would be better. 

> We may need agree on this: In theory, I could have fixed the bug myself quickly, but every project seems to handle this differently, so I stick to comments for now.

Unless there is a very strict timeframe or the solution is hard to explain in English, I prefer fixing my own problems and letting  others fix theirs. Do you have any preferences, @otacke @henriettemoe?

**Review by otacke — APPROVED** [review]
We'll deal with formatting later anyway ...

**Comment by otacke** [comment]
> Unless there is a very strict timeframe or the solution is hard to explain in English, I prefer fixing my own problems and letting others fix theirs. Do you have any preferences, @otacke @henriettemoe?

Never thought about that much before. I go for what is common in the project that I worki in. I think I have a preference for keeping the total time used low regardless of who did what. If this was my own repository and commenting took longer than just adding the line of code as in this case (and it was not a case of repeated negligence or a learning opportunity for someone who doesn't know better yet), I'd do it myself. Here, I am definitely not telling you anything that you would not know ... But if you prefer fixing those small things on your own, that's fine with me :-)

**Pull request merged**